### PR TITLE
Create LinkFree profile for SavvasStephanides

### PR DESCRIPTION
**Pull request by SavvasStephanides**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4442"></a>

### CODE REVIEW

File: data/SavvasStephanides.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/SavvasStephanides",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/savvasstephanides",
+      "icon": "FaGithub"
```
Comment (priyankarpal) [inline data/SavvasStephanides.json:25]: change the icon to `FaLinkedin`

Comment (priyankarpal) [inline data/SavvasStephanides.json:25]: Change the icon to `FaLinkedin`

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Change the icon to `FaLinkedin`

File: data/SavvasStephanides.json (hunk 1 above)
Comment (SavvasStephanides) [reply data/SavvasStephanides.json]: Thank you Priyankar! Didn't notice that! 🙏

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**